modules/nw-sriov-expose-mtu.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/add-pod.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-expose-mtu_{context}"]
+= Exposing MTU for vfio-pci SR-IOV devices to pod
+
+After adding a pod to an additional network, you can check that the MTU is available for the SR-IOV network.
+
+.Procedure
+
+. Check that the pod annotation includes MTU by running the following command:
++
+[source,terminal]
+----
+$ oc describe pod example-pod
+----
+The following example shows the sample output:
++
+[source,yaml]
+----
+"mac": "20:04:0f:f1:88:01",
+       "mtu": 1500,
+       "dns": {},
+       "device-info": {
+         "type": "pci",
+         "version": "1.1.0",
+         "pci": {
+           "pci-address": "0000:86:01.3"
+    }
+  }
+----
+
+. Verify that the MTU is available in `/etc/podnetinfo/` inside the pod by running the following command:
++
+[source,terminal]
+----
+$ oc exec example-pod -n sriov-tests -- cat /etc/podnetinfo/annotations | grep mtu
+----
+The following example shows the sample output:
++
+[source,json]
+----
+k8s.v1.cni.cncf.io/network-status="[{\n    \"name\": \"ovn-kubernetes\",\n    \"interface\": \"eth0\",\n    \"ips\": [\n        \"10.131.0.67\"\n    ],\n    \"mac\": \"0a:58:0a:83:00:43\",\n    \"default\": true,\n    \"dns\": {}\n},{\n    \"name\": \"sriov-tests/sriov-nic-1\",\n    \"interface\": \"net1\",\n    \"ips\": [\n        \"192.168.10.1\"\n    ],\n    \"mac\": \"20:04:0f:f1:88:01\",\n    \"mtu\": 1500,\n    \"dns\": {},\n    \"device-info\": {\n        \"type\": \"pci\",\n        \"version\": \"1.1.0\",\n        \"pci\": {\n            \"pci-address\": \"0000:86:01.3\"\n        }\n    }\n}]"
```

Comment:
Thanks @jldohmann for your feedback. I have applied your suggestion.

Reply:
Could we format this so that it is legible in the output? 

For example, this is the same content formatted so that it can be read easily:
```cmd
k8s.v1.cni.cncf.io/network-status="[{
    \"name\": \"ovn-kubernetes\",
    \"interface\": \"eth0\",
    \"ips\": [
        \"10.131.0.67\"
    ],
    \"mac\": \"0a:58:0a:83:00:43\",
    \"default\": true,
    \"dns\": {}
    },{
    \"name\": \"sriov-tests/sriov-nic-1\",
    \"interface\": \"net1\",
    \"ips\": [
        \"192.168.10.1\"
    ],
    \"mac\": \"20:04:0f:f1:88:01\",
    \"mtu\": 1500,
    \"dns\": {},
    \"device-info\": {
        \"type\": \"pci\",
        \"version\": \"1.1.0\",
        \"pci\": {
            \"pci-address\": \"0000:86:01.3\"
        }
    }
    }]"
```

Reply:
Thanks @aireilly for your feedback. I have applied your suggestion.